lesson_9/index9.html

```diff
@@ -5,47 +5,45 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document HW</title>
+    <title>Resume</title>
 </head>
 <body>
-    <div>
-        <h1 class="line">Iryna Masiuk</h1>
-        <form action="C:/всяке/Robot_Dreams/lesson_6/index6.html">
-            <input type="submit" value="Offer job" name="offer_job">
-        </form>
-        <br>
-        <img src="myphoto.jpg" width="240" height="320">
-        <h3 class="line">Junior frontend developer</h3>
+    <div class="top-block">
+        <div class="text-aline">
```

Comment:
I don't see you use such a class in your .css
why it's needed?

Reply:
I forgot to change the name of the class. I fixed